tests/datasource/gwas_catalog/test_gwas_catalog_curation.py

```diff
@@ -0,0 +1,252 @@
+"""Test GWAS Catalog study curation logic."""
```

Comment:
I'd also rename the module as previously mentioned
```suggestion
"""Test GWAS Catalog study metadata curation logic."""
```